rootfs/ingest/python/qserv/http.py

```diff
@@ -168,8 +195,30 @@ def get(self, url: str,
         _LOG.debug("GET: success")
         return response_json
 
-    def post(self, url: str, payload: Dict[str, Any] = dict(),
-             auth: bool = True, timeout: int = None) -> Dict:
+    def post(
+        self, url: str, payload: Dict[str, Any] = dict(), auth: bool = True, timeout: int = None
```

Comment:
It is not OK to use `dict()` as a default argument value: https://developer.lsst.io/python/style.html#a-mutable-object-must-not-be-used-as-a-keyword-argument-default
Please check all new methods below.